docs/index.rst

```diff
@@ -1,230 +1,232 @@
-3. Contigs and Scaffolds (output of metaSPAdes) are consumed by the *create_agp* task to rename the FASTA header and generate an `AGP format <https://www.ncbi.nlm.nih.gov/assembly/agp/AGP_Specification/>`_ which describes the assembly
+1. The *bbcms* task takes in interleaved FASTQ inputs, performs error correction, and reformats the interleaved FASTQ into two output FASTQ files for paired-end reads for the next tasks. 
+2. The *assy* task performs metaSPAdes assembly.
+3. Contigs and Scaffolds (output of metaSPAdes) are processed by the *create_agp* task to rename the FASTA header and generate an `AGP format <https://www.ncbi.nlm.nih.gov/assembly/agp/AGP_Specification/>`_ which describes the assembly.
 4. The *read_mapping_pairs* task maps reads back to the final assembly to generate coverage information.
-5. The final *make_output* task adds all output files into the specified directory.
+5. The final *make_output* task collects all output files into the specified directory.
+
+**Long Reads**:
+
+In long reads, the workflow uses Flye for assembly, pbmm2 for alignment, Racon for polishing, and minimap2 for read mapping and coverage analysis. The :literal:`.wdl` (Workflow Description Language) file includes six tasks: *combine_fastq*, *assy*, *racon*, *format_assembly*, *map*, and *make_info_file*.
+
+1. The *combine_fastq* task combines the input FASTQ files into a single FASTQ file, which is used as input for polishing and mapping tasks.
+2. The *assy* task takes in the input FASTQ files and performs assembly using Flye.
+3. The *racon* task cleans up the assembled contigs through two rounds of error correction using :literal:`pbmm2` and :literal:`Racon`.
+4. The *format_assembly* task formats the polished assembly using BBTools' :literal:`fungalrelease.sh`, creating release-ready scaffolds and contigs, along with an `AGP format <https://www.ncbi.nlm.nih.gov/assembly/agp/AGP_Specification/>`_ file and a legend file that describes the assembly.
+5. The *map* task maps the input reads back to the final assembly using minimap2 to generate coverage data.
+6. The final *make_info_file* task produces a summary file documenting tool versions, parameters, memory usage, and Docker containers used throughout the workflow.
+
 
 Workflow Availability
 ---------------------
 
-The workflow from GitHub uses all the listed docker images to run all third-party tools.
-The workflow is available in GitHub: https://github.com/microbiomedata/metaAssembly; the corresponding Docker images are available in DockerHub: https://hub.docker.com/r/microbiomedata/spades and https://hub.docker.com/r/microbiomedata/bbtools
+The workflow from GitHub uses all the listed Docker images to run all third-party tools.  
+
+The workflow is available on GitHub: `https://github.com/microbiomedata/metaAssembly`  
+
+The corresponding Docker images are available on DockerHub:
+
+- `https://hub.docker.com/r/microbiomedata/spades`
+- `https://hub.docker.com/r/microbiomedata/bbtools`
```

Comment:
These are for long reads and should be added.
https://hub.docker.com/r/staphb/flye
https://hub.docker.com/r/staphb/racon
https://hub.docker.com/r/staphb/minimap2
https://hub.docker.com/r/staphb/samtools